src/bid/bid.rs

```diff
@@ -1,14 +1,14 @@
 //! Bid data structure
 
+use super::BidGenerationError;
+pub(crate) use crate::bid::StorageBid;
 use crate::score_gen::{compute_score, Score};
 use dusk_plonk::jubjub::{
     AffinePoint, ExtendedPoint, GENERATOR, GENERATOR_NUMS,
 };
 use dusk_plonk::prelude::*;
 use failure::Error;
-use poseidon252::sponge::sponge::sponge_hash;
-pub(crate) mod encoding;
-pub use encoding::StorageBid;
+use poseidon252::{cipher::PoseidonCipher, sponge::sponge::sponge_hash};
```

Comment:
`PoseidonCipher` is not used anywhere in this PR. Why is it imported?

Reply:
Yep, I should remove it, my bad. This was from the previous impl.